assets/src/components/block-navigation/index.js

```diff
@@ -10,7 +10,8 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import { BlockPreviewLabel } from './';
+import { BlockPreviewLabel } from '../';
+import './edit.css';
 
 /**
  * Internal dependencies
```

Comment:
`import { ALLOWED_MOVABLE_BLOCKS } from '../constants';` below should be changed to `import { ALLOWED_MOVABLE_BLOCKS } from '../../constants';` with the new structure.